src/subdomains/staking/domain/entities/staking.entity.ts

```diff
@@ -209,82 +143,17 @@ export class Staking extends IEntity {
     return addresses.every((a) => a === this.withdrawalAddress.address);
   }
 
-  updateBalance(): number {
-    const confirmedDeposits = this.getDepositsByStatus(DepositStatus.CONFIRMED);
-    const confirmedWithdrawals = this.getWithdrawalsByStatus(WithdrawalStatus.CONFIRMED);
-
-    const confirmedDepositsAmount = Util.sumObj(confirmedDeposits, 'amount');
-    const confirmedWithdrawalsAmount = Util.sumObj(confirmedWithdrawals, 'amount');
-
-    this.balance = Util.round(confirmedDepositsAmount - confirmedWithdrawalsAmount, 8);
-
-    // staged balances (staked more than xxx days)
-    const stageOneDeposits = confirmedDeposits.filter((d) => Util.daysDiff(d.created, new Date()) < 2);
-    this.stageOneBalance = Math.max(0, Util.round(this.balance - Util.sumObj(stageOneDeposits, 'amount'), 8));
-
-    const stageTwoDeposits = confirmedDeposits.filter((d) => Util.daysDiff(d.created, new Date()) < 6);
-    this.stageTwoBalance = Math.max(0, Util.round(this.balance - Util.sumObj(stageTwoDeposits, 'amount'), 8));
-
-    return this.balance;
-  }
-
   //*** GETTERS ***//
 
-  getWithdrawal(withdrawalId: number): Withdrawal {
-    const withdraw = this.withdrawals.find((w) => w.id === withdrawalId);
-
-    if (!withdraw) throw new NotFoundException('Withdrawal not found');
-
-    return withdraw;
-  }
-
-  getDraftWithdrawals(): Withdrawal[] {
-    return this.withdrawals.filter((w) => w.status === WithdrawalStatus.DRAFT);
-  }
-
-  getDeposit(depositId: string): Deposit {
-    const deposit = this.deposits.find((w) => w.id === parseInt(depositId));
-
-    if (!deposit) throw new NotFoundException('Deposit not found');
-
-    return deposit;
-  }
-
-  getDepositByPayInTxId(payInTxId: string): Deposit {
-    return this.deposits.find((w) => w.payInTxId === payInTxId);
-  }
-
-  getBalance(): number {
+  get _balance(): number {
     return this.balance;
   }
```

Comment:
What is this for?

Reply:
habit, I usually have encapsulation in entities, and getters for all props (all private). No setters since state changes to be done only via meaningful domain methods. With typeorm this is useless and confusing, I agree, will remove. just accessing class property directly from outside feels disturbing, so... yeah